### PR TITLE
polish for initial submit

### PR DESCRIPTION
**Pull request by henkbirkholz**

### CODE REVIEW

File: draft-ftbs-rats-msg-wrap.md [hunk 1]
```diff
@@ -73,21 +73,24 @@ defined in {{-rats-arch}}.
...
-## Type and Value Array {#type-n-val}
+1.) a CMW using a CBOR or a JSON array ({{type-n-val}})
 
-The CMW described in {{fig-cddl}} is a CDDL array of two elements:
+2.) a CMW based on CBOR tags ({{cbor-tag}}).
 
-* type: a string encoding a media type {{-media-types}} or an
+## CMW Array {#type-n-val}
+
+The CMW Array illustrated in {{fig-cddl}} is composed of two members:
```
Comment (thomas-fossati) [inline draft-ftbs-rats-msg-wrap.md:85]:
```suggestion
The CMW array illustrated in {{fig-cddl}} is composed of two members:
```

**Review by thomas-fossati — COMMENTED** [review]
LGTM; bunch of small editorial comments

**Review by nedmsmith — APPROVED** [review]

File: draft-ftbs-rats-msg-wrap.md [hunk 2]
```diff
@@ -100,15 +103,16 @@ When using CBOR, the value field is serialized as a CBOR bytes string.
 {: #fig-cddl artwork-align="left"
    title="CDDL definition"}
 
-## CBOR Tags derived from CoAP Content Formats {#cbor-tag}
+## CMW CBOR Tags {#cbor-tag}
 
-If a CoAP Content Format exists for the RATS conceptual message, the
+CBOR Tags used as CMW are derived from CoAP Content Format values.
+If a CoAP Content Format exists for a RATS conceptual message, the
```
Comment (hannestschofenig) [inline draft-ftbs-rats-msg-wrap.md:109]:
```suggestion
If a CoAP Content Format exists for a CMW, the
```

Comment (thomas-fossati) [reply draft-ftbs-rats-msg-wrap.md:109]: mmm, it's really the fact that a content format is defined for the top-level "conceptual message" that enables the automatic CBOR tagging.

File: draft-ftbs-rats-msg-wrap.md [hunk 3]
```diff
@@ -73,21 +73,24 @@ defined in {{-rats-arch}}.
...
+## CMW Array {#type-n-val}
+
+The CMW Array illustrated in {{fig-cddl}} is composed of two members:
+
+* type: ether a text string representing a media-type {{-media-types}} or an
   unsigned integer corresponding to a CoAP Content-Format {{-coap}}
 
-* value: the RATS conceptual message serialisation according to the
-  specified type.
+* value: the RATS conceptual message serialized according to the
```
[the remark this answers is not in the document]
Comment (thomas-fossati) [reply draft-ftbs-rats-msg-wrap.md:90]: the CMW is the whole she-bang, not just the value.

**Pull request merged**